packages/app/app/components/FavoriteAlbumsView/index.tsx

```diff
@@ -10,6 +9,31 @@ import Header from '../Header';
 
 import styles from './styles.scss';
 
+type Tracklist = { 
+  uuid:string;
+  artist: { name: string };
+  title: string;
+  duration: number;
+  }
+
+type Album = {
+  id: string;
+  title: string;
+  thumb: string;
+  coverImage: string;
+  artists: { name:string } [];
+  images: Array<string>;
+  genres: Array<string>;
+  tracklist: Array<Tracklist>
+}
+
+type FavoriteAlbumsViewProps = {
+  albums: Array<Album>;
+  removeFavoriteAlbum: React.MouseEventHandler;
+  albumInfoSearch: (albumId: any, releaseType: any, release: any) => Promise<void>;
```

Comment:
albumId and release should be strings instead of any, releaseType is a union of string literals (`'master' | 'release'`)

Reply:
Got it. In hindsight, I used this because of another component `ArtistView/index.tsx` and I believe that must be fixed as well along with this. I will make the consequent changes in both. 

```
type ArtistViewProps = {
  artist: Artist & {
    loading?: boolean;
  };
  isFavorite: boolean;
  addTrackToQueue: (item: any) => Promise<void>;
  artistInfoSearchByName: (artistName: any) => Promise<void>;
  albumInfoSearch: (albumId: any, releaseType: any, release: any) => Promise<void>;

```